extract/extract/extractors/amazon.py

```diff
@@ -42,6 +45,7 @@ def extract_amazon(parsed_page: ParsedPage) -> Optional[Product]:
     Returns:
         Optional[Product]: Valid `Product` object or `None` if extraction failed
     """
+    language = re.sub("www.amazon.", "", urlparse(parsed_page.scraped_page.url).netloc)
```

Comment:
Since the first parameter is compile to RegEx, we should escape the `.` because it' a special character in regex: `"www\.amazon\."`

Reply:
Corrected :)